File: php/Disk.php
<?php
class Disk
{
    private $type;
    private $capacity;
    private $price;

    public function __construct($type = "", $capacity = 0, $price = 0)
    {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    public function getType()
    {
        return $this->type;
    }

    public function setType($type)
    {
        $this->type = $type;
    }

    public function getCapacity()
    {
        return $this->capacity;
    }

    public function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    public function getPrice()
    {
        return $this->price;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function printDisk()
    {
        echo "Disk Type: " . $this->type . "<br/>";
        echo "Disk Capacity: " . $this->capacity . " GB<br/>";
        echo "Disk Price: " . $this->price . "<br/>";
    }
}

File: java/Disk.java

class Disk {
    private String type;
    private int capacity;
    private int price;

    public Disk(String type, int capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    public Disk() {
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public void printDisk() {
        System.out.println("Disk Type: " + this.type);
        System.out.println("Disk Capacity: " + this.capacity + " GB");
        System.out.println("Disk Price: " + this.price);
    }

}
